src/main/webapp/view/transcript/transcript.js

```diff
@@ -149,18 +151,18 @@ function removeBold(transcriptLineLiElement) {
   transcriptLineLiElement.classList.remove(BOLD_FONT_WEIGHT);
 }
 
-/** Returns true if`transcriptLineLiElement` is bolded. */
+/** Returns true if `transcriptLineLiElement` is bolded. */
 function isBolded(transcriptLineLiElement) {
   return transcriptLineLiElement.classList.contains(BOLD_FONT_WEIGHT);
 }
 
 /**
  * Returns true if `currentTimeMs` is within the time range for
- * the current transcript line.
+ * 'transcriptLine'
  */
-function isWithinCurrentTimeRange(currentTimeMs) {
-  return currentTranscriptLine.startTimestampMs <= currentTimeMs &&
-      currentTimeMs <= currentTranscriptLine.endTimestampMs;
+function isWithinTimeRange(currentTimeMs, transcriptLine) {
```

Comment:
currentTimeMs is slightly confusing? What does current time mean. Maybe rename to currentVideoTimeMs.

Reply:
I think this function should work for any time in milliseconds so I think it would be better to not mention video time. Would it be better to update it to `timeMs`?